tensorrt_llm/_torch/auto_deploy/transform/library/export_to_gm.py

```diff
@@ -185,6 +228,17 @@ def _is_child(child: str, parent: str) -> bool:
                     patch_list=self.config.patch_list,
                 )
 
+            # Ensure runtime calls from HF into this exported GraphModule do not fail due to
+            # torch.export's strict input spec checks (e.g., HF passing attention_mask).
+            sub_gm.register_forward_pre_hook(_ad_sanitize_gm_kwargs, prepend=True, with_kwargs=True)
```

Comment:
see my other comment. Ideally we don't do that and can instead adopt our input processor